lib/nailinda_web/router.ex

```diff
@@ -18,6 +18,8 @@ defmodule NailindaWeb.Router do
 
     get "/patient/new", PatientController, :new
     get "/", PageController, :index
+    get "/patient/:id", PatientController, :edit
```

Comment:
how about you use patient resource

Reply:
it shall be refactored once delete has been merged, not on this branch